<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comClass</class>
 <widget class="QWidget" name="comClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>com</string>
  </property>
  <property name="windowIcon">
   <iconset resource="com.qrc">
    <normaloff>:/com/zip.png</normaloff>:/com/zip.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QRadioButton" name="radioButton_zlib">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>zlib</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_quicklz">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>quicklz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_compress">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>compress</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_decompress">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>decompress</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>output</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="com.qrc"/>
 </resources>
 <connections/>
</ui>
